src/core/graph.test.js

```diff
@@ -692,7 +721,11 @@ describe("graph", () => {
             .addEdge(fullyDanglingEdge())
             .removeNode(danglingSrc().address)
             .removeNode(danglingDst().address);
-          const inEdges = g.inEdges(fullyDanglingEdge().dst);
+          const inEdges = g
+            .neighborhood(fullyDanglingEdge().dst, {
```

Comment:
A bunch of these objects can be collapsed onto the same line without Prettier complaining; it becomes more readable that way, IMO, but up to you.